config/routes.rb

```diff
@@ -14,13 +14,10 @@
   post "/api/v1/merchants", to: "api/v1/merchants#create"
 
   delete "/api/v1/merchants/:id", to: "api/v1/merchants#destroy"
-  
-  
   get "/api/v1/items", to: "api/v1/items#index"
   get "/api/v1/items/:id", to: "api/v1/items#show"
   patch "/api/v1/items/:id", to: "api/v1/items#update" 
-
   post "/api/v1/items", to: "api/v1/items#create"
-  get "/api/v1/items?sorted=price", to: "api/v1/items#index"
 
+  get '/api/v1/items/:id/merchant', to: 'api/v1/merchant_items#show'
```

Comment:
endpoint looks great! Did you test with Postman?

Reply:
Seems like it, ran it a couple a times and i dont get any errors

Reply:
Double checked on Postico as well